hnn_core/dipole.py

```diff
@@ -17,7 +17,7 @@ def _hammfilt(x, winsz):
     return convolve(x, win, 'same')
 
 
-def simulate_dipole(net, n_trials=None, record_vsoma=False):
+def simulate_dipole(net, n_trials=None, record_vsoma=False, postproc=True):
```

Comment:
This is one of the lines with a merge conflict due to the somatic currents PR. I'd recommend changing it to:
```py
def simulate_dipole(net, n_trials=None, postproc=True, record_vsoma=False, record_isoma=False):
```